docs/standard/base-types/quantifiers-in-regular-expressions.md

```diff
@@ -18,265 +19,265 @@ ms.assetid: 36b81212-6511-49ed-a8f1-ff080415312f
 |-------------|-----------------|  
-|`\b`|Start at a word boundary.|  
-|`an?`|Match an "a" followed by zero or one "n" character.|  
-|`\b`|End at a word boundary.|  
+|`\b`|Starts at a word boundary.|  
+|`an?`|Matches an `a` followed by zero or one `n` character.|  
+|`\b`|Ends at a word boundary.|  
   
 ### Match Exactly n Times: {n}  
 
  The `{`*n*`}` quantifier matches the preceding element exactly *n* times, where *n* is any integer. `{`*n*`}` is a greedy quantifier whose lazy equivalent is `{`*n*`}?`.  
   
- For example, the regular expression `\b\d+\,\d{3}\b` tries to match a word boundary followed by one or more decimal digits followed by three decimal digits followed by a word boundary. The following example illustrates this regular expression.  
+ For example, the regular expression `\b\d+\,\d{3}\b` tries to match a word boundary followed by one or more decimal digits followed by three decimal digits followed by a word boundary. The following example illustrates this regular expression:
   
  [!code-csharp[RegularExpressions.Quantifiers#4](../../../samples/snippets/csharp/VS_Snippets_CLR/RegularExpressions.Quantifiers/cs/Quantifiers1.cs#4)]
  [!code-vb[RegularExpressions.Quantifiers#4](../../../samples/snippets/visualbasic/VS_Snippets_CLR/RegularExpressions.Quantifiers/vb/Quantifiers1.vb#4)]  
   
- The regular expression pattern is defined as shown in the following table.  
+ The regular expression pattern is defined as shown in the following table:
   
 |Pattern|Description|  
 |-------------|-----------------|  
-|`\b`|Start at a word boundary.|  
-|`\d+`|Match one or more decimal digits.|  
-|`\,`|Match a comma character.|  
-|`\d{3}`|Match three decimal digits.|  
-|`\b`|End at a word boundary.|  
+|`\b`|Starts at a word boundary.|  
+|`\d+`|Matches one or more decimal digits.|  
+|`\,`|Matches a comma character.|  
+|`\d{3}`|Matches three decimal digits.|  
+|`\b`|Ends at a word boundary.|  
   
 ### Match at Least n Times: {n,}  
 
  The `{`*n*`,}` quantifier matches the preceding element at least *n* times, where *n* is any integer. `{`*n*`,}` is a greedy quantifier whose lazy equivalent is `{`*n*`,}?`.  
   
- For example, the regular expression `\b\d{2,}\b\D+` tries to match a word boundary followed by at least two digits followed by a word boundary and a non-digit character. The following example illustrates this regular expression. The regular expression fails to match the phrase `"7 days"` because it contains just one decimal digit, but it successfully matches the phrases `"10 weeks and 300 years"`.  
+ For example, the regular expression `\b\d{2,}\b\D+` tries to match a word boundary followed by at least two digits followed by a word boundary and a non-digit character. The following example illustrates this regular expression. The regular expression fails to match the phrase `"7 days"` because it contains just one decimal digit, but it successfully matches the phrase `"10 weeks and 300 years"`.  
```

Comment:
```suggestion
 For example, the regular expression `\b\d{2,}\b\D+` tries to match a word boundary followed by at least two digits followed by a word boundary and a non-digit character. The following example illustrates this regular expression. The regular expression fails to match the phrase `"7 days"` because it contains just one decimal digit, but it successfully matches the phrases `"10 weeks"` and `"300 years"`.  
```